lib/galaxy/tool_util/verify/asserts/archive.py

```diff
@@ -49,7 +49,7 @@ def _list_from_zip(bytes, path):
     return sorted(lst)
 
 
-def assert_has_archive_member(output_bytes, path, verify_assertions_function, children, all="false", n: int = None, delta: int = 0, min: int = None, max: int = None, negate: bool = False):
+def assert_has_archive_member(output_bytes, path, verify_assertions_function, children, all="false", n: str = None, delta: str = 0, min: str = None, max: str = None, negate: bool = False):
```

Comment:
If they can be int as well this should be a Union, and if the default is None I think you need typing.Optional
```suggestion
def assert_has_archive_member(output_bytes, path, verify_assertions_function, children, all="false", n: Optional[Union[str, int]] = None, delta: Union[str, int] = 0, min: Optional[Union[str, int]] = None, max: Optional[Union[str, int]] = None, negate: bool = False):
```